test/utils/feature.flag.spec.ts

```diff
@@ -1,60 +1,32 @@
-import { describe, expect, test } from '@jest/globals';
+import { describe, expect } from '@jest/globals';
 import { isActiveFeature } from "../../src/utils/feature.flag";
 
-describe("feature flag tests", function() {
-
-  test("should return false if variable is 'false'", function() {
-    const active = isActiveFeature("false");
-    expect(active).toBeFalsy;
-  });
-
-  test("should return false if variable is '0'", function() {
-    const active = isActiveFeature("0");
+describe("feature flag tests", function () {
+  const dataExpectFalse = [
+    'false',
+    'faLse',
+    '0',
+    'off',
+    'oFF',
+    'kdjhskjf',
+  ];
+
+  it.only.each(dataExpectFalse)('should return false if variable is %p', (val) => {
+    const active = isActiveFeature(<any>val);
     expect(active).toBeFalsy;
   });
 
-  test("should return false if variable is ''", function() {
-    const active = isActiveFeature("");
-    expect(active).toBeFalsy;
-  });
-
-  test("should return false if variable is undefined", function() {
-    const active = isActiveFeature(undefined);
-    expect(active).toBeFalsy;
-  });
-
-  test("should return true if variable is random", function() {
-    const active = isActiveFeature("kdjhskjf");
-    expect(active).toBeTruthy;
-  });
-
-  test("should return false if variable is 'off'", function() {
-    const active = isActiveFeature("off");
-    expect(active).toBeFalsy;
-  });
+  const dataExpectTrue = [
+    'true',
+    'trUe',
+    'TRUE',
+    'on',
+    'oN',
+    '1',
+  ];
 
-  test("should return false if variable is 'OFF'", function() {
-    const active = isActiveFeature("OFF");
+  it.only.each(dataExpectTrue)('should return true if variable is %p', (val) => {
+    const active = isActiveFeature(<any>val);
```

Comment:
You do not need `<any>`. Use `any` when strictly necessary

Reply:
there is need casting to `any` because of
>  Argument of type 'string' is not assignable to parameter of type 'FeatureFlag'.

Reply:
Nope, but it is not a blocker